org.scala-ide.sdt.debug/src/org/scalaide/debug/internal/model/ScalaThread.scala

```diff
@@ -330,8 +334,11 @@ private[model] class ScalaThreadActor private(thread: ScalaThread) extends BaseD
       thread.threadRef.resume()
     case DropToFrame(frame) =>
       thread.dropToFrameInternal(frame)
-    case RebindStackFrames =>
-      if (thread.isSuspended) thread.rebindScalaStackFrames()
+    case RebindStackFrames(shouldFireChangeEvent) =>
+      if (thread.isSuspended) {
+        thread.rebindScalaStackFrames()
+        if (shouldFireChangeEvent) thread.fireChangeEvent(DebugEvent.CONTENT)
```

Comment:
It's only to refresh frames when e.g. someone assigned the new value to var. In EE we don't drop frames.